docs/en/v0.4/reference/sql/range.md

```diff
@@ -0,0 +1,332 @@
+# RANGE QUERY
+
+Querying and aggregating data within a range of time is a common query pattern for time series data, such as the `Range selector` in `PromQL`. GreptimeDB supports Range queries in SQL, which is used to summarize time series data into time chunks and aggregate data on time chunks. As part of the `SELECT` statement, Range query can be flexibly combined with SQL to provide more powerful time series data query capabilities in SQL.
+
+## Syntax
+
```

Comment:
The syntax should be something like (just an example):
```sql
SELECT
  RANGE_EXPRESSION(column1) RANGE TIME_INTERVAL,
  ....
FROM table_name
TIME_DURATION BY(column2) FILLING_METHOD
```
Then explain every concepts in the syntax:

- RANGE_EXPRESSION
- RANGE keyword
- TIME_INTERVAL
.....